decomposed/extensions/MDDE_LDM/Profile/BaseClassifierMapping/Templates/mdde_BaseClassifierMapping_XmlExport_Template.xml

```diff
@@ -16,6 +16,11 @@
   .foreach_item(mdde_Mapping_Examples,\n  &lt;Examples&gt;\n,\n  &lt;/Examples&gt;)
     %mdde_Example_XmlExport_Template%
   .next(\n)
+  .comment Add the content of the export extension template
+  .if (%Model.mdde_export_extension_attached%)
```

Comment:
If the extension is attached, but the contents is empty don't we always get a newline now? So don't we need a nested if with:
.if(%.D:mddex_BaseClassifierMapping_Export_Content?%)

Reply:
Actually I was thinking about the question mark at the end in my comment, it should be without to check whether there is contents (on not whether the extended attribute exists)

Reply:
Goldplating

Reply:
Agreed, changed